dyno-jedis/src/main/java/com/netflix/dyno/jedis/DynoJedisClient.java

```diff
@@ -533,6 +533,42 @@ public Long execute(final Jedis client, final ConnectionContext state) throws Dy
         });
     }
 
+    
+    @Override
+    public ScanResult<Map.Entry<String, String>> hscan(final String key, final int cursor) {
```

Comment:
ScanResult<T> is Jedis class (not interface). I'd prefer not to paint ourselves into this corner again, as we did with Response<T>. Client libraries that leak their underlying dependencies but aren't strictly delegating as passthrough quickly get into trouble...

On the other hand, I can appreciate that the Dyno client should be a drop-in replacement (to the extent this makes sense) for Jedis in order for client applications to seamlessly switch between the two. This approach does make Dyno client code more convoluted however, and will tie the Dyno client to specific version(s) of the Jedis client.

Thoughts ?